test/commands/import.spec.js

```diff
@@ -1,214 +1,236 @@
     command.run(['node', `${BIN_DIR}/sphere-import`])
-    command.program.name.should.be.a.Function
-    command.program.name().should.equal('sphere-import')
-    command.program.commands.should.have.lengthOf(0)
-    command.program.options.should.have.lengthOf(6)
-    command.program.options[0].flags.should.equal('-p, --project <key>')
-    command.program.options[1].flags.should.equal('-t, --type <name>')
-    command.program.options[2].flags.should.equal('-f, --from <path>')
-    command.program.options[3].flags.should.equal('-b, --batch <n>')
-    command.program.options[4].flags.should.equal('-c, --config <object>')
-    command.program.options[5].flags.should.equal('--plugin <path>')
-    command.program.should.not.have.property('project')
-    command.program.should.not.have.property('type')
-    command.program.should.not.have.property('from')
-    command.program.should.not.have.property('config')
-    command.program.should.have.property('batch')
-    expect(spy1.args[0][0]).toEqual({ batch: 5 })
-    expect(spy1.args[0][1]).toEqual('type')
-    expect(spy2.args[0][0]).toEqual('Missing required options: type')
+    t.equal(typeof command.program.name, 'function')
+    t.equal(command.program.name(), 'sphere-import')
+    t.equal(command.program.commands.length, 0)
+    t.equal(command.program.options.length, 6)
+    t.equal(command.program.options[0].flags, '-p, --project <key>')
+    t.equal(command.program.options[1].flags, '-t, --type <name>')
+    t.equal(command.program.options[2].flags, '-f, --from <path>')
+    t.equal(command.program.options[3].flags, '-b, --batch <n>')
+    t.equal(command.program.options[4].flags, '-c, --config <object>')
+    t.equal(command.program.options[5].flags, '--plugin <path>')
+    t.notOk(command.program.project)
+    t.notOk(command.program.type)
+    t.notOk(command.program.from)
+    t.notOk(command.program.config)
+    t.ok(command.program.batch)
+    t.deepEqual(spy1.args[0][0], { batch: 5 })
+    t.deepEqual(spy1.args[0][1], 'type')
+    t.equal(spy2.args[0][0], 'Missing required options: type')
+    t.end()
   })
+})
 
-  it('should process stock command', () => {
+test(`ImportCommand
+  should process stock command`, (t) => {
+  before().then((command) => {
     const spy1 = sinon.stub(command, '_stream')
-    const spy2 = sinon.stub(command, '_preProcess', opts => {
-      return command._process(
-        Object.assign(opts, { config: {}, credentials: fakeCredentials }))
-    })
+    const spy2 = sinon.stub(command, '_preProcess', opts => command._process(
+      Object.assign(opts, { config: {}, credentials: fakeCredentials }))
+    )
     command.run(['node', `${BIN_DIR}/sphere-import`,
       '-p', 'foo', '-t', 'stock', '-f', './foo.json'])
-    command.program.project.should.be.equal('foo')
-    command.program.type.should.be.equal('stock')
-    command.program.from.should.be.equal('./foo.json')
-    expect(spy1.args[0][0]).toEqual({
+    t.equal(command.program.project, 'foo')
```

Comment:
In general it's good to provide a message as third param explaining what the assertion does